src/cockpit/bootloader.py

```diff
@@ -0,0 +1,56 @@
+# COCKPIT BOOTLOADER
```

Comment:
This is intriguing... could you document the purpose of this code briefly?

Reply:
I'll mention it in the commit message, but I don't want to add a comment, since this will show up in the output of `python -c`.  That's why that one comment is there.